ui/src/dashboards/utils/protoboardSuggestion.ts

```diff
@@ -0,0 +1,164 @@
+// Libraries
+import _ from 'lodash'
+// Utils
+import {proxy} from 'src/utils/queryUrlGenerator'
+// Types
+import {Source, Protoboard} from 'src/types'
+
+interface SeriesObject {
+  measurement: string
+  host: string
+}
+
+export const getSuggestedProtoboards = async (
+  source: Source,
+  protoboards: Protoboard[]
+) => {
+  const hosts = await getHosts(source)
+
+  if (!hosts) {
+    return []
+  }
+
+  const newHosts = await addAppsToHosts(source, hosts, protoboards)
+
+  const suggestedProtoboards = _.reduce(
+    newHosts,
+    (acc, host) => {
+      if (host.apps && host.apps.length > 0) {
```

Comment:
nitpick but you could just do `host.apps && host.apps.length`

Reply:
thanks!